### ISSUE TITLE
remove vue-play from dependencies

### ISSUE THREAD

**Issue by achtan** [SSENSE/vue-carousel#351]
why is vue-play listed in dependencies ?
after installing vue-carousel i got ton of unwanted packages...
````
$ yarn add vue-carousel
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
[4/4] 📃  Building fresh packages...
success Saved lockfile.
success Saved 14 new dependencies.
├─ dom-walk@0.1.1
├─ global@4.3.2
├─ highlight.js@9.13.1
├─ min-document@2.19.0
├─ process@0.5.2
├─ query-string@4.3.4
├─ strict-uri-encode@1.1.0
├─ uid@0.0.2
├─ vue-carousel@0.16.1
├─ vue-play@3.2.1
├─ vue-router@2.8.1
├─ vue-slim-modal@0.3.0
├─ vuex-router-sync@4.3.2
└─ vuex@2.5.0
````

**Comment by quinnlangille** [comment]
`vue-play` is a dependency of the package, we use it for our sandbox environment

**Comment by quinnlangille** [comment]
Just published the fix `vue-carousel@0.16.2` 🚀
